neue-kriminalpolitik.csl

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="de-DE">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Neue Kriminalpolitik (Deutsch)</title>
+    <title-short>NK</title-short>
+    <id>http://www.zotero.org/styles/neue-kriminalpolitik</id>
+    <link href="http://www.zotero.org/styles/neue-kriminalpolitik" rel="self"/>
+    <link href="https://www.nk.nomos.de/fileadmin/nk/doc/NK_Autorenhinweise_2015.pdf" rel="documentation"/>
+    <author>
+      <name>AlexVonB</name>
+      <uri>https://github.com/AlexVonB</uri>
+    </author>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <issn>0934-9200</issn>
+    <updated>2021-08-07T14:15:12+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="authors-inline">
+    <names variable="author" font-style="italic">
+      <name form="short" font-style="normal" delimiter="/" name-as-sort-order="all"/>
```

Comment:
Would it make sense to have substitute elements here for editor and title so that things like edited volumes and webpages look better? (Check the automatically generated sample citations to see what I mean.) Same for author-bib

Reply:
You are right, I'll have to test some things with the bibliography, I'll let you know when this is ready. Thanks!

Reply:
Now it is resolved, thank you for your feedback; I did not think of books without authors or websites :)